src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -48,6 +56,22 @@ export class ExamChecklistComponent implements OnChanges, OnInit, OnDestroy {
         const startedTopic = this.examChecklistService.getStartedTopic(this.exam);
         this.websocketService.subscribe(startedTopic);
         this.websocketService.receive(startedTopic).subscribe(() => (this.numberOfStarted += 1));
+        if (this.exam && this.exam.course) {
+            this.quizPoolService.find(this.exam.course.id!, this.exam.id!).subscribe({
+                next: (response: HttpResponse<QuizPool>) => {
+                    this.quizPool = response.body!;
+                },
+                error: (error: HttpErrorResponse) => {
+                    if (error.status === 404) {
```

Comment:
How is `404` not an error and doesn't cause an alert? What could cause an `404`-error that should be handled by ignoring it?

Reply:
In this case, it is not an error from the client perspective because it is possible that an exam (e.g. old exam) doesn’t have a quiz pool entity. This error is not ignored but it is handled by initializing a new quiz pool entity.